chain/network/src/types.rs

```diff
@@ -288,7 +288,7 @@ pub enum NetworkRequests {
     /// Message for a chunk endorsement, sent by a chunk validator to the block producer.
     ChunkEndorsement(AccountId, ChunkEndorsement),
     /// Message from chunk producer to set of chunk validators to send state witness part.
-    PartialEncodedStateWitness(Vec<(AccountId, PartialEncodedStateWitness)>),
+    PartialEncodedStateWitness((AccountId, PartialEncodedStateWitness)),
```

Comment:
nit: this doesn't need to be a tuple, it can just be two values

```suggestion
    PartialEncodedStateWitness(AccountId, PartialEncodedStateWitness),
```